src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/Cas1SpaceBookingEntity.kt

```diff
@@ -0,0 +1,36 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity
+
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.stereotype.Repository
+import java.time.LocalDate
+import java.time.OffsetDateTime
+import java.util.UUID
+import javax.persistence.Entity
+import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.ManyToOne
+import javax.persistence.Table
+
+@Repository
+interface Cas1SpaceBookingRepository : JpaRepository<Cas1SpaceBookingEntity, UUID> {
+  fun findByPremisesIdAndPlacementRequestId(premisesId: UUID, placementRequestId: UUID): Cas1SpaceBookingEntity?
+}
+
+@Entity
+@Table(name = "cas1_space_bookings")
+data class Cas1SpaceBookingEntity(
```

Comment:
Can we please tag all @ManyToOne relationships as lazy. We've had lots of performance issues on Spring Boot 3 and the prevailing advice seems to be that 'eager' is a bad default for @ManyToOne because it ends up generating huge 'cascading' queries. We can always change to eager once we've established how we're using these entities.

Reply:
I've updated these. Weird that the default wasn't changed given the poor performance introduced in the new version of Spring.

Reply:
Yes, it's a strange one but i guess as it's in the JPA spec it's hard to change without breaking existing deployments :(